doc/source/io.rst

```diff
@@ -3278,13 +3278,22 @@ External Compatibility
 producing loss-less round trips to pandas objects. For external
 compatibility, ``HDFStore`` can read native ``PyTables`` format
 tables. It is possible to write an ``HDFStore`` object that can easily
-be imported into ``R`` using the ``rhdf5`` library. Create a table
-format store like this:
+be imported into ``R`` using the
+``rhdf5`` library (`Package website`_). Create a table format store like this:
+
+.. _package website: http://www.bioconductor.org/packages/release/bioc/html/rhdf5.html
 
      .. ipython:: python
 
+        np.random.seed(1)
+        df_for_r = pd.DataFrame({"first": np.random.rand(100),
+                                 "second": np.random.rand(100),
+                                 "class": np.random.randint(0, 2, (100,))},
+                                 index=range(100))
+        print(df_for_r.head())
```

Comment:
don't need print statements, just do: ``df_for_r.head()``, it will show the output